### PR TITLE
MachineLearningService: changes for serviceProvisionedRG on workspace

### PR DESCRIPTION
**Pull request by DeMale**
creation

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-01-01/machineLearningServices.json [hunk 1]
```diff
@@ -1461,6 +1461,11 @@
           "type": "boolean",
           "description": "The flag to signal HBI data in the workspace and reduce diagnostic data collected by the service",
           "default": false
+        },
+        "serviceProvisionedResourceGroup": {
+          "type": "string",
+          "description": "The name of the scoped resource group created by workspace RP in customer subscription",
```
Comment (zhaomuzhi) [inline specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-01-01/machineLearningServices.json]: Change to "The name of the scoped resource group created by workspace RP in customer subscription if the workspace is cmk workspace"?

**Review by zhaomuzhi — APPROVED** [review]

**Review by noorabani — APPROVED** [review]

**Comment by NullMDR** [comment]
@DeMale Please fix the prettier following https://aka.ms/AA6h31t

**Comment by NullMDR** [comment]
@DeMale Is this PR ready for merge?

**Comment by DeMale** [comment]
Hi,
Yes, the PR is ready to merge.
Thanks.

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Review by PhoenixHe-NV — APPROVED** [review]

**Pull request merged**